packages/ui-extensions/src/surfaces/admin/api/customer-segmentation-template/customer-segmentation-template.ts

```diff
@@ -18,11 +18,27 @@ type TemplateCategory =
   | 'purchaseBehaviour'
   | 'location';
 
+type UnitSystem = 'IMPERIAL_SYSTEM' | 'METRIC_SYSTEM';
+
+interface Location {
+  name: string;
+  latitudeDecimalDegrees?: number;
```

Comment:
Note that lat/long are optionals as we might not have them for invalid addresses.

Reply:
The general Shop api reference https://shopify.dev/docs/api/admin-rest/2022-10/resources/shop mentions only `lattitude/longitude`. Ok with both, but just mentioning it in case we are ok with a general key instead.